tests/utils.go

```diff
@@ -2999,21 +3002,42 @@ func LoggedInFedoraExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	}
 	b := append([]expect.Batcher{
 		&expect.BSnd{S: "\n"},
-		&expect.BExp{R: "login:"},
-		&expect.BSnd{S: "fedora\n"},
-		&expect.BExp{R: "Password:"},
-		&expect.BSnd{S: "fedora\n"},
-		&expect.BExp{R: "$"},
+		&expect.BSnd{S: "\n"},
+		&expect.BCas{C: []expect.Caser{
+			&expect.Case{
+				// Using only "login: " would match things like "Last failed login: Tue Jun  9 22:25:30 UTC 2020 on ttyS0"
```

Comment:
just imo , the comments that were around here are good to have,
i understand sometimes its outdated, but meanwhile it can teach people stuff
```
// The following is a switch case. First case matched wins.
// If we ever see a prompt ("$ "), we succeed.
// If all the retries (Rt) of all the cases are exhausted, we fail.
// No need to use the timeout version (BCasT), ExpectBatch already has one.
```

Reply:
The code is not a manual, such text makes reading the code harder and with time becomes incorrect.
One can add such text to the commit message.

Reply:
better than nothing then imo